packages/services/f-http/src/httpVerbs.js

```diff
@@ -0,0 +1,7 @@
+const METHOD_GET = 'GET';
```

Comment:
Should we just call them `GET` and `POST` instead of `METHOD_XXXXX`?

Reply:
I wanted to do this originally; but DELETE is a keyword in Javascript :D

Reply:
I thought `delete` was, but didn't know `DELETE` was! 🤯

Reply:
Hmm, good point; let me check